components/rankings/rankingsUtils.js

```diff
@@ -1,13 +1,16 @@
 import moment from 'moment'
 
-const getDateFromMessage = async message => {
-  const year = moment().format('YYYY')
+const getDateFromMessage = async ({ msg }) => {
+  let year = moment().format('YYYY')
   let month = moment().format('M')
-  const monthFromMessage = message.msg.replace(/[^0-9]+/g, '')
+  const [, monthFromMessage, yearFromMessage] = msg.trim().split(' ')
 
   if (await isValidMonth(monthFromMessage)) {
```

Comment:
Não é necessário ser uma promise aqui, já que é apenas um teste de regex
```suggestion
  if (isValidMonth(monthFromMessage)) {
```

Reply:
Feito!